docs/serving/autoscaling/scale-bounds.md

```diff
@@ -7,6 +7,10 @@ type: "docs"
 
 To apply upper and lower bounds to autoscaling behavior, you can specify scale bounds in both directions.
 
+You can also specify the initial target scale of a revision after creation, with either its cluster-wide flag or annotation.
+
+If both `minScale` annotation and initial scale (global or per-revision) are specified, and `minScale` is greater than the calculated initial scale, the initial scale will be ignored.
```

Comment:
```suggestion
If both `minScale` and initial scale annotations (global or per-revision) are specified, and `minScale` is greater than the calculated initial scale, the initial scale will be ignored.
```

Reply:
I just updated this to make it more explicit (instead of saying "global or per-revision" here)